features/git-ship/current_branch/on_feature_branch/without_open_changes/same_user_different_email.feature

```diff
@@ -0,0 +1,34 @@
+Feature: git ship: shipping a feature branch where the main committer is the same user
```

Comment:
What about `Feature: git ship: shipping a feature branch where the main committer is the same account using a different email address`?